lib/omniauth/strategies/openid_connect.rb

```diff
@@ -228,7 +228,7 @@ def pkce_authorize_params(verifier)
       private
 
       def fetch_key
-        @fetch_key ||= parse_jwk_key(::OpenIDConnect.http_client.get_content(client_options.jwks_uri))
+        @fetch_key ||= parse_jwk_key(::OpenIDConnect.http_client.get(client_options.jwks_uri).body)
```

Comment:
I wonder if we should be using something inside the `openid_connect` gem instead of reaching directly into the HTTP client, but perhaps that's for another day.